app/distributions/api.py

```diff
@@ -109,21 +108,7 @@ def attributions(request: HttpRequest, lang: LanguageCode | None = None):
     lang_to_use = get_language(lang, request.headers)
 
     responses = [attribution_to_response(model, lang_to_use) for model in models]
-    return AttributionListSchema(items=responses)
-
-
-def dataset_to_response(model: Dataset) -> DatasetSchema:
-    """
-    Transforms the given model into a response object.
-    """
-    return DatasetSchema(
-        id=str(model.id),
-        slug=model.slug,
-        created=model.created,
-        updated=model.updated,
-        provider_id=str(model.provider.id),
-        attribution_id=str(model.attribution.id),
-    )
+    return {"items": responses}
```

Comment:
:thinking: Fine with me here because the decorator defines the return type and it makes it shorter.

Still, the same concerns as mentioned above apply:
1. Less explicit: I have to check the decorator to see what is returned ultimately.
2. No type checking: Not sure if mypy would alert me here if `items` was something different from a list.
3. Less coding assistance: Anything can go into that dict.

So fine if you want to use a dict here. But not a fan :wink: